awesomeConfig.cmake

```diff
@@ -352,6 +357,33 @@ foreach(file ${AWESOME_CONFIGURE_FILES})
                    ESCAPE_QUOTES
                    @ONLY)
 endforeach()
+
+set(AWESOME_CONFIGURE_NO_COV_FILES
+    ${awesome_c_configure_files}
+    ${awesome_lua_configure_files}
+)
+
+if(NOT DO_COVERAGE)
+    foreach(file ${AWESOME_CONFIGURE_NO_COV_FILES})
+        configure_file(${SOURCE_DIR}/${file}
+                    ${BUILD_DIR}/${file}
+                    ESCAPE_QUOTES
+                    @ONLY)
+    endforeach()
+else()
+    foreach(file ${AWESOME_CONFIGURE_NO_COV_FILES})
+        configure_file(${SOURCE_DIR}/${file}
+                    ${BUILD_DIR}/${file}
+                    COPYONLY)
+    endforeach()
+
+    # There is no way to avoid that one
+    configure_file(${SOURCE_DIR}/lib/awful/util.lua
+                    ${BUILD_DIR}/lib/awful/util.lua
+                    ESCAPE_QUOTES
+                    @ONLY)
```

Comment:
Ah:
```lua
--- Get the path to the directory where themes are installed.
-- @return A string with the requested path with a slash at the end.
function util.get_themes_dir()
    return "@AWESOME_THEMES_PATH@" .. "/"
end

--- Get the path to the directory where our icons are installed.
-- @return A string with the requested path with a slash at the end.
function util.get_awesome_icon_dir()
    return "@AWESOME_ICON_PATH@" .. "/"
end
```

Should/can we move this to a separate file?
Or is it not too much of a hack to handle the whole of `util.lua` in a special way here?